v2/cmd/integration-test/integration-test.go

```diff
@@ -13,6 +13,12 @@ import (
 	sliceutil "github.com/projectdiscovery/utils/slice"
 )
 
+type TestCaseInfo struct {
+	DisableOn []string
```

Comment:
What about turning `DisableOn` into a callback like `func() bool` so that additional logic can be implemented if necessary